.eslintrc

```diff
@@ -25,221 +32,152 @@
-    "yoda": [0, "never"],
 
     /* Strict Mode */
     "strict": [0, "global"],
 
     /* Variables */
-    "no-catch-shadow": 0,
-    "no-delete-var": 1,
     "no-label-var": 1,
-    "no-shadow-restricted-names": 1,
-    "no-shadow": 0,
-    "no-undef-init": 1,
-    "no-undefined": 0,
     "no-unused-vars": [
       1,
       {
         "vars": "local",
         "args": "none"
       }
     ],
-    "no-use-before-define": 0,
-
-    /* Node.js */
-    "handle-callback-err": 0,
-    "no-mixed-requires": 1,
-    "no-new-require": 1,
-    "no-path-concat": 1,
-    "no-process-exit": 1,
-    "no-restricted-modules": [1, ""], // add any unwanted Node.js core modules
-    "no-sync": 1,
 
-    /* Stylistic Issues */
-    "array-bracket-spacing": [0, "never"],
-    "brace-style": [
-      0,
-      "1tbs",
-      {
-        "allowSingleLine": true
-      }
-    ],
     "camelcase": [
       0,
       {
         "properties": "always"
       }
     ],
-    "comma-spacing": [
-      0,
-      {
-        "before": false,
-        "after": true
-      }
-    ],
-    "comma-style": [1, "last"],
-    "comma-dangle": 0,
-    "computed-property-spacing": 0,
-    "consistent-this": 0,
-    "eol-last": 0,
-    "func-names": 0,
-    "func-style": 0,
-    "indent": [0, 4, { "SwitchCase": 1 }],
-    "key-spacing": [
-      0,
-      {
-        "beforeColon": false,
-        "afterColon": true
-      }
-    ],
-    "linebreak-style": 0,
-    "max-nested-callbacks": [0, 3],
-    "new-cap": 0, // see Babel section
-    "new-parens": 1,
-    "newline-after-var": 0,
-    "no-array-constructor": 1,
-    "no-continue": 0,
-    "no-inline-comments": 0,
-    "no-lonely-if": 0,
-    "no-mixed-spaces-and-tabs": 0,
-    "no-multiple-empty-lines": [
-      0,
-      {
-        "max": 1
-      }
-    ],
-    "no-nested-ternary": 0,
+    "no-array-constructor": "error",
     "no-new-object": 1,
-    "no-spaced-func": 0,
-    "no-ternary": 0,
-    "no-trailing-spaces": 0,
-    "no-underscore-dangle": 0,
-    "no-unneeded-ternary": 1,
-    "object-curly-spacing": 0, // see Babel section
-    "one-var": [0, "never"],
-    "padded-blocks": [0, "never"],
-    "quote-props": [0, "as-needed"],
-    "quotes": [0, "single"],
-    "semi-spacing": [
-      1,
-      {
-        "before": false,
-        "after": true
-      }
-    ],
-    "semi": [1, "always"],
-    "sort-vars": 0,
-    "space-after-keywords": 0,
-    "space-before-blocks": [0, "always"],
-    "space-before-function-paren": [0, "never"],
-    "space-in-parens": [0, "never"],
-    "space-infix-ops": 0,
-    "space-unary-ops": 0,
-    "spaced-comment": [0, "always"],
-    "wrap-regex": 0,
+    "no-unneeded-ternary": 1 /* ECMAScript 6 */,
+    "prefer-const": "error"
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:react-hooks/recommended",
+        "plugin:@typescript-eslint/recommended"
```

Comment:
here you also need a `plugin:@typescript-eslint/eslint-recommended` so js eslint rules are converted to ts version

Reply:
actually, my bad `plugin:@typescript-eslint/recommended` will activate it automatically but IMO doesn't hurt to add it explicitly, your call

Reply:
I'd prefer to be explicit, will add it